torch/_inductor/ir.py

```diff
@@ -3830,13 +3843,20 @@ def codegen(self, wrapper):
             kernel, configs, self.kwargs
         )
 
+        args = self.codegen_kwargs()
+        if V.graph.cpp_wrapper:
+            # in C++ wrapper, we don't pass constexpr args, as they don't
+            # get added as parameters to the PTX code compiled from the
+            # user-defined Triton kernel (only non-constexpr args do)
+            args = self._remove_constexpr_args(args)
```

Comment:
The above
```
kernel, configs = self.get_kernel_and_configs()
```
already extracted the kernel for you, so just kernel to this function instead of duplicating that logic